### PR TITLE
[Core] Logout Session

### PR DESCRIPTION
**Pull request by tehAgitto**
# Pull Request summary:

```python
session = create_vManageSession(
    url=base_url,
    username=username,
    password=password,
    port=port
)
print(session.about())

with create_vManageSession(url=base_url, username=username, password=password, port=port) as session:
    print(session.about())
```

`create_vManageSession` works now as standalone function and collaborate with `with` keyword.

### CODE REVIEW

**Comment by tehAgitto** [comment]
Closes #82

**Comment by tehAgitto** [comment]
fixes #233

**Comment by tehAgitto** [comment]
fixes #233

**Review by cicharka — DISMISSED** [review]

File: vmngclient/session.py [hunk 1]
```diff
@@ -344,6 +344,20 @@ def get_virtual_session_id(self, tenant_id: str) -> str:
         response = self.post(url_path)
         return response.json()["VSessionId"]
 
+    def logout(self) -> vManageResponse:
+        if self.api_version >= Version("20.12"):
+            return self.post("/logout")
+        else:
+            return self.get("/logout")
+
+    def __enter__(self):
```
Comment (sbasan) [inline vmngclient/session.py:353]: Do we really need to override `__enter__` and `__exit__`?
I think the better way i just to override `close` method https://requests.readthedocs.io/en/latest/_modules/requests/sessions/#Session.close

File: vmngclient/session.py [hunk 2]
```diff
@@ -344,6 +344,20 @@ def get_virtual_session_id(self, tenant_id: str) -> str:
...
+        return self
+
+    def __exit__(self, *exc):
+        super().__exit__()
```
Comment (sbasan) [inline vmngclient/session.py:357]: This closes transport adapters (not sure if it has any noticable effect) but sending logout request after this statement will possibly reopen them.
https://docs.python-requests.org/en/v1.2.3/api/#requests.adapters.HTTPAdapter

**Review by sbasan — CHANGES_REQUESTED** [review]

**Review by cicharka — APPROVED** [review]

**Review by sbasan — APPROVED** [review]
lgtm

**Pull request merged**